Opdrachten.js

```diff
@@ -0,0 +1,166 @@
+// Opdracht 1 - Array Methoden
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+
+const tvName = inventory.map((inventoryItem) => {
+    return inventoryItem.name;
+})
+console.log(tvName);
+
+// Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+const tvSoldOut = inventory.filter((inventoryItem) => {
+    return inventoryItem.originalStock - inventoryItem.sold === 0;
+})
```

Comment:
Dit zou je nog wat korter kunnen schrijven
```suggestion
const tvSoldOut = inventory.filter(inventoryItem => inventoryItem.originalStock === inventoryItem.sold);
```